ui/StatusQ/src/StatusQ/Components/StatusToastMessage.qml

```diff
@@ -38,7 +38,7 @@ import StatusQ.Core.Theme 0.1
 Control {
     id: root
     width: 343
-    height: !!secondaryText ? 68 : 48
+    height: !!secondaryText || title.lineCount > 1 ? 68 : 48
```

Comment:
Is this necessary at all, specifying the `height`? Control should do that automatically

Reply:
I have't seen the initial designs, but might be necessary to comply with design. Reading the code, I understand the height is static and has two states: Small and Large (48 and 68). And it made sense to me because the toaster content can have more than 2 variations and it could result in an array of possible heights for the toaster.

Reply:
No big deal... just wondering it will most probably overflow when the user selects some big font size